include/openssl/crypto.h

```diff
@@ -376,7 +376,7 @@ int CRYPTO_memcmp(const void * in_a, const void * in_b, size_t len);
 # define OPENSSL_INIT_ENGINE_CAPI            0x00002000L
 # define OPENSSL_INIT_ENGINE_PADLOCK         0x00004000L
 # define OPENSSL_INIT_ENGINE_AFALG           0x00008000L
-# define OPENSSL_INIT_reserved_internal      0x00010000L
+# define OPENSSL_INIT_BASE_ONLY              0x00010000L
```

Comment:
Does the documentation need to be updated?

Reply:
Good point.